src/frontend/komponenter/Fagsak/Behandlingsresultat/EndretUtbetalingAndelSkjema.tsx

```diff
@@ -288,53 +271,23 @@ const EndretUtbetalingAndelSkjema: React.FunctionComponent<IEndretUtbetalingAnde
-                        value={
-                            skjema.felter.søknadstidspunkt.verdi !== null
-                                ? skjema.felter.søknadstidspunkt.verdi
-                                : undefined
-                        }
-                        label={<Label>Søknadstidspunkt</Label>}
-                        placeholder={DatoformatNorsk.DATO}
-                        onChange={(dato?: ISODateString) =>
-                            skjema.felter.søknadstidspunkt.validerOgSettFelt(dato)
-                        }
-                        erLesesvisning={erLesevisning}
+                    <Datovelger
+                        felt={skjema.felter.søknadstidspunkt}
+                        label={'Søknadstidspunkt'}
+                        visFeilmeldinger={skjema.visFeilmeldinger}
+                        readOnly={erLesevisning}
+                        kanKunVelgeFortid
                     />
                 </Feltmargin>
 
                 {skjema.felter.avtaletidspunktDeltBosted.erSynlig && (
                     <Feltmargin>
-                        <StyledFamilieDatovelger
-                            {...skjema.felter.avtaletidspunktDeltBosted.hentNavBaseSkjemaProps(
-                                skjema.visFeilmeldinger
-                            )}
-                            feil={
-                                !!skjema.felter.avtaletidspunktDeltBosted.feilmelding &&
-                                skjema.visFeilmeldinger
-                            }
-                            value={
-                                skjema.felter.avtaletidspunktDeltBosted.verdi !== null
-                                    ? skjema.felter.avtaletidspunktDeltBosted.verdi
-                                    : undefined
-                            }
-                            label={<Label>Avtale om delt bosted</Label>}
-                            placeholder={DatoformatNorsk.DATO}
-                            onChange={(dato?: ISODateString) =>
-                                skjema.felter.avtaletidspunktDeltBosted.validerOgSettFelt(dato)
-                            }
-                            erLesesvisning={erLesevisning}
+                        <Datovelger
+                            felt={skjema.felter.avtaletidspunktDeltBosted}
+                            label={'Avtale om delt bosted'}
+                            visFeilmeldinger={skjema.visFeilmeldinger}
+                            readOnly={erLesevisning}
                         />
-                        {skjema.felter.avtaletidspunktDeltBosted.feilmelding &&
-                            skjema.visFeilmeldinger && (
-                                <StyledErrorMessage>
-                                    {skjema.felter.avtaletidspunktDeltBosted.feilmelding}
-                                </StyledErrorMessage>
-                            )}
```

Comment:
Feilmeldinger vises inne i datovelger og trenger ikke å ha egen custom feilmelding for avtaletidspunkt delt bosted